api/src/pcapi/core/users/api.py

```diff
@@ -775,6 +775,19 @@ def add_comment_to_user(user: models.User, author_user: models.User, comment: st
     db.session.commit()
 
 
+def _get_booking_credit(booking: bookings_models.Booking) -> Decimal:
+    # Get only partial incidents
+    booking_finance_incidents = [
+        booking_finance_incident
+        for booking_finance_incident in booking.incidents
+        if booking_finance_incident.is_partial
+        and booking_finance_incident.incident.status == finance_models.IncidentStatus.VALIDATED
+    ]
```

Comment:
la comprehension list sur 5 ligne, je pense qu'on gagnerait en lisibilité en la déroulant avec une boucle for standard.